gravitee-am-identityprovider/gravitee-am-identityprovider-inline/src/main/resources/schemas/schema-form.json

```diff
@@ -34,7 +34,8 @@
             "title": "Password",
             "description": "Password",
             "type" : "string",
-            "widget": "password"
+            "widget": "password",
+            "sensitive": true
```

Comment:
The password field of the inline provider is marked as sensitive but it is not filtered.
The REST response contains the clear value. Probably because the users defiend in the inline provider configuration are inside an array

Reply:
@leleueri 
> The password filtering isn't working for inline provider
Since the inline providers purpose is for `ease-to-use` and testing the solution, this is hasn't been done.

> The Certificate plugins contains two password fields and should be handled by the SensitiveProxy class

I will add it

> The gravitee-am-identityprovider-kerberos EE module should be updated to mark password as sensitive (like LDAP idp). As for the inline provider, the kerberos sensistive element will be in an inner object.

I will also look into it

> Finally, we should take into account the comment of YoranSys about the update. Indeed, If I remove a single 
character an update is performed. Maybe simply test if the new value only contains the same character used to  ​mask the sensitive data.

My thought was to check  
```java
fieldValue.trim().matches("^(\*+)$") 
```
but wanted to discuss first

Reply:
:+1: